src/main/java/com/teamabnormals/abnormals_core/core/utils/NetworkUtil.java

```diff
@@ -1,6 +1,11 @@
 package com.teamabnormals.abnormals_core.core.utils;
 
 import com.teamabnormals.abnormals_core.common.world.storage.tracking.IDataManager;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.screen.*;
```

Comment:
Don't import all of the screen package, just import what's needed.

Reply:
damn intellij